src/Flysystem/S3/composer.json

```diff
@@ -16,6 +16,8 @@
         }
     ],
     "require": {
+        "php": "^7.2.5",
+        "async-aws/core": "^0.1 || ^0.2",
```

Comment:
The core package is not needed, right? We never use that one directly.

Reply:
we use it for exception and return typehint.
composer-require-checker tell me